0000-getting-started-cmake.md

```diff
@@ -133,6 +131,8 @@ More details in [Environments](/documentation/0400-environments)
 $> cmake-re -S . -B build/ -DCMAKE_TOOLCHAIN_FILE=environments/linux.cmake
 ```
 
-This will launch the container on your machine and execute the build inside it, while caching the build execution.
+This will launch the container on your machine and execute the build inside it. Build artifacts will be cached.
+
+Alternatively you can run the build on your *bare metal* host by adding `--host` to the command line.
```

Comment:
nope please remove this it's important to not offer all possibility here, it makes products difficult to grasp in my eyes and --host defeats the hermetic builds which is the value here

Reply:
Agreed, we can remove it, it was just content that was previously all in a single sentence with the hint that follows. It read:

> You can get the build to rerun on any changes and check your test outputs with `--monitor` and `--run-test all`, alternatively if you don't want to run the containerized build but still use caching on your bare metal host, you can use : `--host`.

Reply:
Yeah sorry saw that, I think we should remove it. :-)